fades/main.py

```diff
@@ -356,7 +357,13 @@ def go():
 
     # store usage information
     usage_manager.store_usage_stat(venv_data, venvscache)
-    if child_program is None:
+    if args.module:
+        logger.debug(
+            "Executing module %r", args.module)
+        module_option = ["-m"] + args.module.split()
+        cmd = [python_exe] + python_options + module_option
+        p = subprocess.Popen(cmd)
+    elif child_program is None:
```

Comment:
I think this is the wrong place to insert this new behaviour. 

There's a big `if` down there, first part is running interactive Python shell (with `interactive=True`), the second is running a child program somehow. This new `--module` behaviour belongs to the `else` part of this `if`.

IIUC, it would have a `args.executable`, but you should double check it.

Thanks!